app/views/pages/home.html.erb

```diff
@@ -3,3 +3,7 @@
     Find me in app/views/pages/home.html.erb
     This is the home page. It is still a work in progress.
 </p>
+
+<%= link_to "Sign Out", destroy_user_session_path, method: :DELETE %>
```

Comment:
Style as button?

Reply:
That would be the plan, but I figured we would probably put the sign out button on the header or sidebar anyways